src/packer.pkr.hcl

```diff
@@ -67,30 +67,40 @@ source "amazon-ebs" "windows" {
     volume_size           = 8
     volume_type           = "gp3"
   }
+
   ami_name                    = "windows-commando-hvm-${local.timestamp}-x86_64-ebs"
   ami_regions                 = var.ami_regions
   associate_public_ip_address = true
-  encrypt_boot                = false
-  instance_type               = "t2.large"
+
   // kms_key_id                  = var.build_region_kms
-  launch_block_device_mappings {
-    delete_on_termination = true
-    device_name           = "/dev/xvda"
-    encrypted             = true
-    volume_size           = 8
-    volume_type           = "gp3"
-  }
-  region = var.build_region
   // region_kms_key_ids = var.region_kms_keys
+  communicator  = "winrm"
+  encrypt_boot  = false
+  instance_type = "t2.large"
+
+  region = var.build_region
+
   skip_create_ami = var.skip_create_ami
   source_ami      = data.amazon-ami.windows.id
 
-  communicator   = "winrm"
+  # Many Linux distributions are now disallowing the use of RSA keys,
+  # so it makes sense to use an ED25519 key instead.
+  temporary_key_pair_type = "ed25519"
+
   user_data_file = "src/winrm_bootstrap.txt"
-  winrm_username = "Administrator"
+
+  winrm_insecure = true
   winrm_timeout  = "20m"
   winrm_use_ssl  = true
-  winrm_insecure = true
+  winrm_username = "Administrator"
+
+  launch_block_device_mappings {
+    delete_on_termination = true
+    device_name           = "/dev/xvda"
+    encrypted             = true
+    volume_size           = 8
+    volume_type           = "gp3"
+  }
```

Comment:
These code blocks should be alphabetized as well.  For example, since this one starts with "l" I think it goes right after `instance_type`.

Reply:
Resolved in commit 6c06e46.